reverse/reverse.py

```diff
@@ -39,4 +39,24 @@ def contains(self, value):
         return False
 
     def reverse_list(self, node, prev):
-        pass
+        if node is None:
+            return None
+        # elif node.get_next() is None:
+        #     return node.value
+        else:   #reformatted to decrease run tme
+            curr_node = node
+
+            while curr_node:
+                self.add_to_head(curr_node.value)
+                curr_node = curr_node.get_next()
```

Comment:
What an efficient solution! I admit I was skeptical about the logic at first, but it checks out!